tests/fuzz/StandaloneFuzzTargetInit.c

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+#include <qpid/dispatch/alloc_pool.h>
+#include <qpid/dispatch/log.h>
+#include "libFuzzingEngine.h"
+
+
+
+
```

Comment:
This file should still have the empty implementation of ```LLVMFuzzerInitialize(argc, argv)```. It is meant to be a (possibly) modified version of the file supplied by either libfuzzer or AFL (I don't remember which one!).

This is just for consistency in building the static library used for regression testing - in theory this file isn't needed any longer for this project.

Ultimately the two file used to build this library should be identical to ones you copied from qpid proton.

Reply:
Thanks @astitcher I have now pushed up a commit to this PR which makes the two files, `tests/fuzz/StandaloneFuzzTargetInit.c` and `tests/fuzz/StandaloneFuzzTargetMain.c` identical to the files in proton. Please take a look.